builtin/settingtypes.txt

```diff
@@ -158,6 +158,229 @@ enable_hotbar_mouse_wheel (Hotbar: Enable mouse wheel for selection) bool true
 #    Requires: keyboard_mouse
 invert_hotbar_mouse_wheel (Hotbar: Invert mouse wheel direction) bool false
 
+#    Key for moving the player forward.
```

Comment:
Might an additional `Requires: keyboard_mouse` be needed for these settings?

Reply:
I am not sure. These keybinding settings only appear in the "Keyboard and mouse" tab which already requires `keyboard_mouse`.

Also the `key` setting type only makes sense in the context of keyboard and mouse control, so for future changes (e.g. SSCSMs) where we would have keybinding settings outside of the "Keyboard and mouse" tab we should consider automatically adding the `keyboard_mouse` requirement to these settings (so that mods will not need to specify the requirement explicitly).

Reply:
https://github.com/luanti-org/luanti/pull/15791/commits/37ce4039617231789582cbdac8e49d5cabaf29dd and https://github.com/luanti-org/luanti/pull/15791/commits/4cc1521c5bec7d33a939839a8b5068f292f6d6f1

Reply:
https://github.com/luanti-org/luanti/pull/15791/commits/5069fc736179f91f626653334afc15715514d25a